UnityProject/Assets/Scripts/Items/Bureaucracy/PaperBin.cs

```diff
@@ -16,7 +16,13 @@ public class PaperBin : NetworkBehaviour, ICheckedInteractable<HandApply>, IServ
 		private int paperCount;
 
 		[SyncVar (hook = nameof(SyncStoredPen))]
-		private GameObject storedPen;
+		private NetworkIdentity _storedPenID;
```

Comment:
Why the underscore?

Reply:
Ryder suggested it

Reply:
It probably was copilot. Underscore for privates isn't our code style.